dawn/package.json

```diff
@@ -28,54 +28,55 @@
     "electron": "./node_modules/.bin/electron"
   },
   "devDependencies": {
-    "babel-cli": "^6.18.0",
-    "babel-core": "^6.5.2",
-    "babel-loader": "^6.2.2",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-react": "^6.3.13",
-    "better-npm-run": "0.0.7",
+    "babel-cli": "^6.22.2",
+    "babel-core": "^6.22.1",
+    "babel-loader": "^6.2.10",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.22.0",
+    "better-npm-run": "0.0.14",
     "chai": "^3.5.0",
-    "electron": "^1.3.5",
-    "eslint": "^2.13.0",
-    "eslint-config-airbnb": "^9.0.1",
-    "eslint-loader": "^1.3.0",
-    "eslint-plugin-import": "^1.8.1",
-    "eslint-plugin-jsx-a11y": "^1.5.3",
-    "eslint-plugin-react": "^5.2.2",
+    "eslint": "^3.14.1",
+    "eslint-config-airbnb": "^14.0.0",
+    "eslint-loader": "^1.6.1",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^3.0.2",
+    "eslint-plugin-react": "^6.9.0",
     "minimist": "^1.2.0",
-    "mocha": "^3.0.2",
-    "redux-devtools": "^3.3.1",
-    "redux-devtools-dock-monitor": "^1.1.1",
-    "redux-devtools-filter-actions": "^1.2.0",
-    "redux-devtools-log-monitor": "^1.0.11",
-    "webpack": "^1.12.13"
+    "mocha": "^3.2.0",
+    "webpack": "^1.14.0"
   },
   "dependencies": {
-    "babel-plugin-transform-object-rest-spread": "^6.8.0",
-    "babel-plugin-transform-regenerator": "^6.9.0",
-    "babel-polyfill": "^6.9.1",
-    "brace": "^0.7.0",
+    "babel-plugin-transform-object-rest-spread": "^6.22.0",
+    "babel-plugin-transform-regenerator": "^6.22.0",
+    "babel-polyfill": "^6.22.0",
+    "brace": "^0.9.1",
+    "electron": "^1.4.15",
     "electron-json-storage": "^1.0.0",
-    "immutable": "^3.7.6",
+    "immutable": "^3.8.1",
     "json-loader": "^0.5.4",
     "keymirror": "^0.1.1",
-    "lodash": "^4.3.0",
-    "mousetrap": "^1.5.3",
-    "numeral": "^1.5.3",
-    "object-assign": "^4.0.1",
+    "lodash": "^4.17.4",
+    "mousetrap": "^1.6.0",
+    "numeral": "^2.0.4",
+    "object-assign": "^4.1.1",
     "protobufjs": "^5.0.1",
-    "react": "^0.14.5",
-    "react-ace": "^3.0.0",
-    "react-bootstrap": "^0.28.1",
-    "react-dom": "^0.14.5",
-    "react-immutable-proptypes": "^1.7.0",
-    "react-joyride": "^1.1.1",
-    "react-redux": "^4.4.5",
-    "redux": "^3.5.2",
+    "react": "^15.4.2",
+    "react-ace": "^4.1.1",
+    "react-bootstrap": "^0.30.7",
+    "react-dom": "^15.4.2",
+    "react-immutable-proptypes": "^2.1.0",
+    "react-joyride": "^1.10.0",
+    "react-redux": "^5.0.2",
+    "redux": "^3.6.0",
+    "redux-devtools": "^3.3.2",
```

Comment:
Linter again, though I personally believe it is better to leave dev-tools for easier debugging on a student-by-student basis. May put under #35.